### ISSUE TITLE
Add notices in the WordPress admin about CORS.

### ISSUE THREAD

**Issue by shulard** [mautic/mautic-wordpress#74]
On the wordpress.org support forum, I got multiple questions about tracking errors. Most of the times it's related to :

* mixed-content : tracker is served over HTTP but the website is HTTPS ;
* script not loaded : tracker script can't load.

I think that adding the ability to test that everything is alright in the settings panel can be a great addition. Though ?

**Comment by robwent** [comment]
I think people are also having issues using dynamic content as cors headers need to be added on the wordpress site to allow the Mautic domain (apparently that's the fix).

It's not something I've used myself but it's come up a couple of times in the slack channels.

Since it's not something a lot of people seem to use, maybe a warning or help/troubleshooting section for this would be ok.

**Comment by shulard** [comment]
Yes, you are totally right. I think that we can parse website header to check if the mautic tracker is enabled or not. Also we can perform some basic checks to ensure that everything will be right with the current setup.

I'll add this to the roadmap.
